### PR TITLE
Return the bounds dictionary in debug mode

### PR DESCRIPTION
**Pull request by guillermoaguilar**
This will allow to do the priors plot and the marginals plot with the correct range. These two plots only work in debug mode.

The plotting routines have been adjusted accordingly to use this new returned dict.

### CODE REVIEW

File: psignifit/psigniplot.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from typing import Union, List
 
+import warnings
```
Comment (pberkes) [inline psignifit/psigniplot.py:4]: nit: the import should be with the standard library import block (one line up)

**Review by pberkes — APPROVED** [review]
LGTM, tiny comment that can be ignored

File: psignifit/psigniplot.py (hunk 1 above)
Comment (guillermoaguilar) [reply psignifit/psigniplot.py:4]: out of curiosity, is there a guideline for that? order of imports? I never know.

Comment (pberkes) [reply psignifit/psigniplot.py:3]: yes, from PEP8:
"""

Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

You should put a blank line between each group of imports.
"""

Comment (guillermoaguilar) [reply psignifit/psigniplot.py:3]: thank you darling.

**Pull request merged**